operators/pkg/forge/containers.go

```diff
@@ -110,20 +112,33 @@ func DeploymentSpec(instance *clv1alpha2.Instance, environment *clv1alpha2.Envir
 // PodSpec forges the pod specification for X-VNC based container instance,
 // conditionally includes the "myDrive" sidecar for standard mode environments.
 func PodSpec(instance *clv1alpha2.Instance, environment *clv1alpha2.Environment, opts *ContainerEnvOpts) corev1.PodSpec {
+	driveMountPath := MyDriveDefaultMountPath
+	var initContainers []corev1.Container
+
+	if environment.ContainerStartupOptions != nil {
+		if environment.ContainerStartupOptions.ContentPath != "" {
+			driveMountPath = environment.ContainerStartupOptions.ContentPath
+		}
+		if environment.ContainerStartupOptions.SourceZipPURL != "" {
+			initContainers = []corev1.Container{ZipDownloaderInitContainer(environment.ContainerStartupOptions, opts)}
```

Comment:
Would define an InitContainers() function, as for the volumes. In general, I would avoid as much as possible to add logic to the top forging functions, but keep it in appropriate functions. Otherwise, those functions will continue to keep growing every time the logic is extended, becoming unmanageable.